INN/umbrella-sfpublicpress
# Homepage projects widget

## Why



Builds the projects widget, resolving #19 with the exception of the series icon output from series metadata.

## Changes

This pull request makes the following changes:

- adds the widget: choose 5 series. As few as 2 are supported, since 3, 4, and 5, can be set to "none"
- adds styles for that widget in the main stylesheet, so that that widget can be used anywhere


## Testing/Questions

Features that this PR affects:

- Widgets admin
- Homepage, if configured correctly


Questions that need to be answered before merging:

- [ ] Does this need a link to Largo's docs on creating series landing pages?
- [x] Why doesn't the series landing page featured image output? This is the major bug in this PR. 

Steps to test this PR:

1. To the Homepage Bottom widget area, add the SFPP Projects List widget. Set it to use a bunch of different series. Make sure that the series chosen as the first series in the widget has a series landing page with a featured image.
